CHANGELOG.D/157.bugfix

```diff
@@ -0,0 +1 @@
+Speedup image build by Kaniko downgrade to v1.1.0 (COPY layers caching), replace /workspace context dir to /kaniko_context.
```

Comment:
so the newer versions don't use COPY layers caching? interesting, why. Because they had a reason, so we might face the problems in future.

Reply:
here's an issue: https://github.com/GoogleContainerTools/kaniko/issues/1357
briefly: it is cheaper to execute COPY then to download and extract it as a cached layer if this layer contains a lot of data